CMakeLists.txt

```diff
@@ -273,7 +273,7 @@ cmake_dependent_option(
 mark_as_advanced(GOOGLE_CLOUD_CPP_ENABLE_REST)
 
 # Prevent users for setting incompatible flags.
-if (GOOGLE_CLOUD_CPP_ENABLE_STORAGE AND NOT GOOGLE_CLOUD_CPP_ENABLE_REST)
+if (storage IN_LIST GOOGLE_CLOUD_CPP_ENABLE AND NOT GOOGLE_CLOUD_CPP_ENABLE_REST)
```

Comment:
just asking, do we need this to be a
```
((GOOGLE_CLOUD_CPP_ENABLE_STORAGE OR storage IN_LIST GOOGLE_CLOUD_CPP_ENABLE) AND NOT GOOGLE_CLOUD_CPP_ENABLE_REST)
```

or does `(storage IN_LIST GOOGLE_CLOUD_CPP_ENABLE)` imply `GOOGLE_CLOUD_CPP_ENABLE_STORAGE` ?

Reply:
Does not matter what the value of `GOOGLE_CLOUD_CPP_ENABLE_STORAGE` is.  The storage library is compiled if and only if `storage IN_LIST GOOGLE_CLOUD_CPP_ENABLE`.